x-pack/legacy/plugins/security/public/views/management/edit_role/components/privileges/kibana/space_aware_privilege_section/privilege_space_table.tsx

```diff
@@ -187,8 +187,10 @@ export class PrivilegeSpaceTable extends Component<Props, State> {
           const effectivePrivilege = effectivePrivileges[record.spacesIndex];
           const basePrivilege = effectivePrivilege.base;
 
-          const isAllowedCustomizations =
-            allowedPrivileges[record.spacesIndex].base.privileges.length > 1;
+          const featureEntries = Object.values(allowedPrivileges[record.spacesIndex].feature);
+          const isAllowedCustomizations = featureEntries.some(entry => {
```

Comment:
Without this change, if I gave the user global all and then a custom set of privileges at a space granting access to ML, it'd show "All" for the space specific privileges:

![all](https://user-images.githubusercontent.com/627123/60926562-c7d72480-a25b-11e9-9368-70c83b1f8b33.gif)